front/app/containers/Authentication/typings.ts

```diff
@@ -8,7 +8,7 @@ import { SuccessAction } from './SuccessActions/actions';
 import { getStepConfig } from './useSteps/stepConfig';
 
 export interface ModalProps {
-  setModalOpen?: (bool: boolean) => void;
+  onToggleModal?: (opened: boolean) => void;
```

Comment:
From the use cases, we could make this required until we get scenarios when it is not needed

```suggestion
  onToggleModal: (opened: boolean) => void;
```